README.md

```diff
@@ -194,6 +200,90 @@ Detailed logs and results are available on Neoload Web. To get the url of the cu
 neoload logs-url                        # The URL to the test in Neoload Web
 ```
 
+### The test-results vs. report subcommands
+
+The 'test-results' subcommand is intended for direct operational queries against high-level API data.
+
+The 'report' subcommand is intended to simplify not only common data exporting needs, but also provide
+ templating capabilities over a standard, correlated data model. In contrast to the test-results
+ subcommand, 'report' can be used to generate as well as transform test result data.
+
+### Exporting Transaction CSV data
```

Comment:
Is would be great to have a list of "builtin" templates. I didn't find it.

Reply:
0ac6fe6 Updated subcommand default usage to print standard help AND extended help including optional argument descriptions and a few examples.